docs/getting_started/install_ez_platform.md

```diff
@@ -312,7 +240,7 @@ Search in the shop front end requires that you have either Solr or Elasticsearch
     ELASTICSEARCH_DSN=http://localhost:9200
     ```
 
-## Install [[= product_name =]]
+### Create a database
```

Comment:
`composer ezplatform-install` does not seem to exists at the moment.
Q: will it be added, or do we need to spell out `php bin/console ezplatform:install clean`, `php bin/console ezplatform:graphql:generate-schema`?

Reply:
The command will be restored as `php bin/console ibexa:install`